deepa/Schedule/ScheduleView.swift

```diff
@@ -37,8 +81,73 @@ struct ScheduleView: View {
     }
 }
 
-struct ScheduleView_Previews: PreviewProvider {
-    static var previews: some View {
-        ScheduleView()
+struct ListContent: View {
+    @State var title: String
+    @State var speakers: Array<Speaker>
+    @State var startAt: Date
+    @State var endAt: Date
+    
+    static let dateFormat: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "E, MMM dd HH:mm"
+        return formatter
+    }()
+    
+    var body: some View {
+        HStack {
+            SpeakersImage(speakers: speakers)
+            
+            VStack(alignment: .leading, spacing: 2) {
+                Text(title)
+                    .font(.title3)
+                    .fontWeight(.bold)
+                    .foregroundColor(.orange)
+                
+                if speakers.count == 1 {
+                    Text(speakers[0].name)
+                        .font(.subheadline)
+                } else {
+                    Text(concatSpeakerName(speakers: speakers, requireCompany: false))
+                        .font(.subheadline)
+                }
+                
+                HStack(spacing: 0) {
+                    Text(startAt, formatter: Self.dateFormat)
+                    Text(" - ")
+                    Text(endAt, formatter: Self.dateFormat)
+                }.font(.caption)
+                    .fontWeight(.light)
+            }
+        }
+    }
+}
+
+extension View {
+    func concatSpeakerName(speakers: Array<Speaker>, requireCompany: Bool) -> String {
+        var speakersList: Array<String> = []
+        if requireCompany {
+            for speaker in speakers {
+                speakersList.append("\(speaker.name) (\(speaker.company ?? ""))")
+            }
+        } else {
+            for speaker in speakers {
+                speakersList.append(speaker.name)
+            }
+        }
+        let speakers = speakersList.joined(separator: " & ")
+        return speakers
+    }
+    
+    func getURL2022(speakerImageURL: String) -> String {
+        if speakerImageURL.count > 0 {
+            var url = speakerImageURL
+            for (index, char) in "2022.".enumerated() {
```

Comment:
LOL this is no no .... we should fix the backend value instead and not hard coding it